install/install_test.go

```diff
@@ -0,0 +1,71 @@
+package install
+
+import (
+	"gopkg.in/check.v1"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (b *InstallSuite) TestTail(c *check.C) {
+	cases := []struct {
+		given  []*corev1.Pod
+		expect []*corev1.Pod
+	}{
+		{
+			given: []*corev1.Pod{
+				{ObjectMeta: v1.ObjectMeta{Name: "Foo"}}},
```

Comment:
Why use capitals here? I find this a bit disorienting, please use all-lower-case.